src/components/HeaderNavMenu.jsx

```diff
@@ -1,16 +1,21 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
+import firebase from 'firebase';
 import { makeStyles, Link as MuiLink, Grid } from '@material-ui/core';
-import LoginButton from './LoginButton';
+import Button from '@material-ui/core/Button';
+import ProfileMenu from './ProfileDropdown';
 
 const useStyles = makeStyles((theme) => ({
   menu: {
-    alignItems: 'center',
+    alignItems: 'right',
     justifyContent: 'center',
   },
   menuItem: {
     color: theme.palette.text.secondary,
   },
+  avatarMenu: {
```

Comment:
Hmm, better name might be `authItem`. `avatarMenu` makes it sound like you're styling the ProfileMenu itself.

Reply:
Ok